contrib-viz/02.contrib-viz.ipynb

```diff
@@ -0,0 +1,819 @@
+      "text/plain": [
+       "plot without title"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Set figure dimensions\n",
+    "width = 5.55\n",
+    "height = 7\n",
+    "options(repr.plot.width=width, repr.plot.height=height)\n",
+    "\n",
+    "gg = month_df %>%\n",
+    "  dplyr::filter(diff_type %in% c('words_added', 'words_deleted')) %>%\n",
+    "  dplyr::arrange(desc(count), desc(diff_type)) %>%\n",
+    "  ggplot2::ggplot(ggplot2::aes(x=year_month, y = username, size = count, color=diff_type)) +\n",
+    "  ggplot2::geom_point(stroke=1) +\n",
+    "  #ggplot2::scale_y_discrete(limits=usage_df$username, position = 'top', labels=y_labels, name=NULL) #+\n",
+    "  ggplot2::scale_x_discrete(limits = year_month_breaks, labels=year_month_labeler, name=NULL) +\n",
+    "  ggplot2::scale_size_area(max_size = 3.8, name='word count') +\n",
```

Comment:
Because we are only showing additions, we could remove the green circle and `added` and change `word count` to `words added`.  I agree with @ajlee21's suggestion to log-scale the size of the circle marker if we can.

Reply:
These suggestions are done in the new version I'll push. But log scaling did not seem to work, it made the dots look very uniform. I agree it would be a more intuitive measure but can't seem to get it to visualize well.